src/aap_eda/api/filters/team.py

```diff
@@ -0,0 +1,29 @@
+#  Copyright 2023 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import django_filters
+
+from aap_eda.core.models import Team
+
+
+class TeamFilter(django_filters.FilterSet):
+    name = django_filters.CharFilter(
+        field_name="name",
+        lookup_expr="istartswith",
+        label="Filter by team name.",
+    )
+
+    class Meta:
+        model = Team
+        fields = ["name"]
```

Comment:
Can you easily add a filter for `organization` as a related field? The would be the obvious thing to add.

Also, maybe description would be desired to be filterable? Both cases, for org and team.

Reply:
Would it be ok to do filter by organization_id or do we wanna do it by organization_name? In the team list view, we just return the organization_id, not the whole object.

Reply:
I would just leave it at the organization_id for now. It gives the UI an option if they need it.